playbooks/integreatly_post_install.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: post-install tasks
```

Comment:
This will need to login to openshift as a first step to support job isolation

Reply:
Rather than logging in we create an in memory host group of the master and run the tasks there.

Reply:
Ah ok, thanks.